src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -56,11 +59,13 @@
 
     /**
      * Test body.
+     * @todo: #120 inline constant to test(s)
```

Comment:
@westonized we need to provide more description here. Imagine you are completely new person in project and you receive just this text as a task